test/package-check-failure.yml

```diff
@@ -0,0 +1,8 @@
+- tasks:
+  - name: install ansible
+    yum: name=ansible state=latest
+
+  - name: install ansible-lint
+    pip: name=ansible-lint
+    args:
+      state: latest
```

Comment:
I was doing this for total testing coverage of any use cases so we know that ansible-lint is picking up any kind of formatting people are doing. I can add a third test for the situation you mentioned.